pkg/systemd/services/services.jsx

```diff
@@ -567,7 +572,7 @@ class ServicesPage extends React.Component {
             unit.shortId = unit.Id.substring(0, unit.Id.length - 8);
     }
 
-    updateProperties(props, path) {
+    updateProperties(props, path, updateFileState = false) {
```

Comment:
Please no spaces around `=` for default arguments

Reply:
Oh this is forced by eslint:
/home/kkoukiou/repos/cockpit/pkg/systemd/services/services.jsx
  568:50  error  Operator '=' must be spaced  space-infix-ops

Reply:
Nope - they don't support this distinction  - sorry :( 
https://github.com/eslint/eslint/issues/7489

Reply:
Nevermind then :)